### DISCUSSION TITLE
Markdig.Extensions.CustomContainers.CustomContainer Not implemented.

### DISCUSSION THREAD

**Discussion by marcostudios** [ppy/osu#22454] [Q&A] [answered]
I was reading #help and I clicked https://osu.ppy.sh/wiki/en/Beatmap/Guest_difficulty and saw this error.

![osu_2023-01-29_01-40-58](https://user-images.githubusercontent.com/5209299/215297806-0bd1ac59-7683-42bb-9b2f-d9f054b53f5c.jpg)

**Comment by Joehuu** [comment, marked as answer]
We have this already tracked as an issue: https://github.com/ppy/osu/issues/14614.
